libs/designsystem/testing-base/src/lib/components/mock.chart.component.ts

```diff
@@ -25,6 +26,7 @@ export class MockChartComponent {
   @Input() data: ChartDataset[] | number[];
   @Input() dataLabels: string[] | string[][];
   @Input() customOptions: ChartOptions;
+  @Input() dataLabelOptions: ChartDataLabelOptions;

```

Comment:
Just for info, I've created an issue to fix the dataLabel terminology confusion. I hope I get around to fix it soon! https://github.com/kirbydesign/designsystem/issues/1905